framework/Simulation.py

```diff
@@ -713,6 +723,8 @@ def run(self):
       subprocess.call(args=remoteRunCommand["args"],
                       cwd=remoteRunCommand.get("cwd", None),
                       env=remoteRunCommand.get("env", None))
+      self.raiseADebug('Submitted in queque! Shutting down Jobhandler!')
```

Comment:
queque should be queue, and it might be useful to add args to this debug. (Something like:
``` self.raiseADebug('Submitted in queue! Shutting down Jobhandler! with'+str(remoteRunCommand["args"]))```

Reply:
done